client/state/plugins/installed/selectors-ts.ts

```diff
@@ -321,3 +322,89 @@ export const getSiteObjectsWithoutPlugin = createSelector(
 	( state: AppState ) => [ getAllPluginsIndexedByPluginSlug( state ), getSitesItems( state ) ],
 	( state: AppState, siteIds: number[], pluginSlug: string ) => [ pluginSlug, ...siteIds ].join()
 );
+
+export function getStatusForPlugin( state: AppState, siteId: number, pluginId: string ) {
+	if ( typeof state.plugins.installed.status[ siteId ] === 'undefined' ) {
+		return false;
+	}
+	if ( typeof state.plugins.installed.status[ siteId ][ pluginId ] === 'undefined' ) {
+		return false;
+	}
+	const status = state.plugins.installed.status[ siteId ][ pluginId ];
+	return Object.assign( {}, status, { siteId: siteId, pluginId: pluginId } );
```

Comment:
We could shorten this and make it more comprehensible by avoiding declaration of a constant that is used just once, plus using spread, and object property shorthand:

```suggestion
	return {
		...state.plugins.installed.status[ siteId ][ pluginId ],
		siteId,
		pluginId
	};
```